dashboard/config/honeybadger.yml

```diff
@@ -0,0 +1,7 @@
+---
+api_key: "#{CDO.dashboard_honeybadger_api_key}"
+
+exceptions.ignore:
+  <%  %w(Sinatra StorageApps DynamoTable PropertyBag).each do |class_name| %>
+  - <%= class_name + "::NotFound" %>
+  <% end %>
```

Comment:
Why are we ignoring these exceptions? Do we want to stop ignoring them at some point?